### PR TITLE
RELEASE - 4.1 release notes

### PR DESCRIPTION
**Pull request by fenryka**
First attempts

### CODE REVIEW

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -1,3 +1,163 @@
+.. contents::
+
+Release notes for Corda 4.1
+===========================
+
+.. _release_notes_v4_1:
+
+Normally this document would start with a comment about weather or not you'd been recently domiciled under some solidified mineral material in relation to the news
```
Comment (gendal) [inline docs/source/release-notes.rst]: whether

File: docs/source/release-notes.rst [hunk 2]
```diff
@@ -1,3 +1,163 @@
...
+of Corda Enterprise 4. Alas, we made that joke when we shipped the first release of Corda after the release of Corda Enterprise 3.0, so the thunder has been
+stolen and repeating ourselves would be terribly gauche. For those who didn't hear that news, lots of information can be found
+`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
+`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_. However, we're not here for that old news.
+
+What we *are* here for is to announce that Corda 4.1 is released today bringing parity with Corda Enterprise 4.0.
```
Comment (gendal) [inline docs/source/release-notes.rst]: parity in what way?  functional parity? bug-for-bug?

File: docs/source/release-notes.rst (hunk 1 above)
Comment (gendal) [inline docs/source/release-notes.rst]: suspect we'll end up toning this intro paragraph down... but let's leave it for _now_ :)

File: docs/source/release-notes.rst [hunk 3]
```diff
@@ -1,3 +1,163 @@
...
+
+Unlike previous point releases of Corda we haven't added any new features, what we have done is learned from all of the work done to bring fourth
```
Comment (gendal) [inline docs/source/release-notes.rst]: forth

File: docs/source/release-notes.rst [hunk 4]
```diff
@@ -1,3 +1,163 @@
...
+* Transactions created in V3 cannot be verified in V4 if any of the state types were included in "depended upon" CorDapps which were not attached to the transaction [`CORDA-2692 <https://r3-cev.atlassian.net/browse/CORDA-2692>`_]
+* Prevent a node re-registering with the doorman if it did already and the node "state" has not been erased [`CORDA-2647 <https://r3-cev.atlassian.net/browse/CORDA-2647>`_]
+* The cert hierarchy diagram for C4 is the same as C3.0 but I thought we changed it between C3.1 and 3.2? [`CORDA-2604 <https://r3-cev.atlassian.net/browse/CORDA-2604>`_]
+* Windows build fails with `FileSystemException` in `TwoPartyTradeFlowTests` [`CORDA-2363 <https://r3-cev.atlassian.net/browse/CORDA-2363>`_]
+* `Cash.generateSpend` cannot be used twice to generate two cash moves in the same tx [`CORDA-2162 <https://r3-cev.atlassian.net/browse/CORDA-2162>`_]
+* FlowException thrown by session.receive is not propagated back to a counterparty
+* invalid command line args for corda result in 0 exit code
+* Windows build fails on TwoPartyTradeFlowTests
+* CE4 performance below CE3
+* Check that we're not calling
```
Comment (gendal) [inline docs/source/release-notes.rst]: ?

File: docs/source/release-notes.rst [hunk 5]
```diff
@@ -1,3 +1,163 @@
...
+Normally this document would start with a comment about weather or not you'd been recently domiciled under some solidified mineral material in relation to the news
+of Corda Enterprise 4. Alas, we made that joke when we shipped the first release of Corda after the release of Corda Enterprise 3.0, so the thunder has been
+stolen and repeating ourselves would be terribly gauche. For those who didn't hear that news, lots of information can be found
+`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
+`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_. However, we're not here for that old news.
+
+What we *are* here for is to announce that Corda 4.1 is released today bringing parity with Corda Enterprise 4.0.
+
+Unlike previous point releases of Corda we haven't added any new features, what we have done is learned from all of the work done to bring fourth
+Corda Enterprise 4 and addressed over 150 bugs and usability issues.
```
Comment (gendal) [inline docs/source/release-notes.rst]: "As such, we recommend you upgrade from Corda 4.0 to Corda 4.1 as soon as possible."

**Review by gendal — COMMENTED** [review]
mostly typos.  suspect the fun police will ask for the intro to be changed before this goes out though :)

File: docs/source/release-notes.rst (hunk 4 above)
Comment (fenryka) [reply docs/source/release-notes.rst]: these were all auto generated from the summaries in Jira

**Comment by fenryka** [comment]
@gendal we should ignore the fun police :)

**Comment by gendal** [comment]
I *am* the fun police :-p

File: docs/source/release-notes.rst [hunk 6]
```diff
@@ -1,3 +1,167 @@
+.. contents::
+
+Release notes for Corda 4.1
+===========================
+
+.. _release_notes_v4_1:
+
+Its been a little under 3 1/2 months since the release of Corda 4.0 and all of the brand new features that added to the powerful suite
+of tools Corda offers. Now, following the release of Corda Enterprise, we are proud to release Corda 4.1, brining over 150 fixes
```
Comment (gendal) [inline docs/source/release-notes.rst]: give a version to CE (4.0).  also s/brining/bringing

File: docs/source/release-notes.rst [hunk 7]
```diff
@@ -1,3 +1,167 @@
...
+and documentation updates to bring stability and quality of life improvements to those developing on the Corda platform.
+
+Normally this document would have started with a comment about whether or not you'd been recently domiciled under some solidified mineral material
+regarding the release of Corda Enterprise 4. Alas, we made that joke when we shipped the first release of Corda after Enterprise 3.0 shipped,
+so the thunder has been stolen and repeating ourselves would be terribly gauche. For those not aware this event occurred, lots of information can be found
+`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
+`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_.
+
+Corda 4.1 brings the lessons and bug fixes discovered during the process of build Enterprise back to the open source community. As mentioned above
```
Comment (gendal) [inline docs/source/release-notes.rst]: during the process of building and shipping Enterprise 4.0

File: docs/source/release-notes.rst [hunk 8]
```diff
@@ -1,3 +1,167 @@
...
+there are over 150 fixes and tweaks here. Additionally, the two have been tested together to ensure smooth operation as well as parity of
+features, as problem we understand and recognised that existed between Corda 3 and Corda Enterprise 3. That confusion was our fault and the release
+of Corda 4.1 to bring clarity is one of those lessons we've learned to make your lives as developers on the Corda platform easier, no matter which
+flavour you're using.
```
Comment (gendal) [inline docs/source/release-notes.rst]: Maybe remove this: "Additionally, the two have been tested together to ensure smooth operation as well as parity of
features, as problem we understand and recognised that existed between Corda 3 and Corda Enterprise 3. That confusion was our fault and the release of Corda 4.1 to bring clarity is one of those lessons we've learned to make your lives as developers on the Corda platform easier, no matter which flavour you're using." ?   My thinking: if we get it right, people won't notice... but if we get it wrong, they'll point at this and ask us what we were thinking!

**Review by gendal — COMMENTED** [review]
minor comments

File: docs/source/release-notes.rst (hunk 6 above)
Comment (fenryka) [reply docs/source/release-notes.rst]: done

File: docs/source/release-notes.rst (hunk 8 above)
Comment (fenryka) [reply docs/source/release-notes.rst]: re-worded to draw out how testing mixed nbetworks should be easier

File: docs/source/release-notes.rst [hunk 9]
```diff
@@ -1,3 +1,165 @@
...
+
+Its been a little under 3 1/2 months since the release of Corda 4.0 and all of the brand new features that added to the powerful suite
+of tools Corda offers. Now, following the release of Corda Enterprise 4.0, we are proud to release Corda 4.1, bringing over 150 fixes
+and documentation updates to bring stability and quality of life improvements to those developing on the Corda platform.
+
+Normally this document would have started with a comment about whether or not you'd been recently domiciled under some solidified mineral material
+regarding the release of Corda Enterprise 4.0. Alas, we made that joke when we shipped the first release of Corda after Enterprise 3.0 shipped,
+so the thunder has been stolen and repeating ourselves would be terribly gauche. For those not aware this event occurred, lots of information can be found
+`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
+`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_.
```
Comment (gendal) [inline docs/source/release-notes.rst]: Trying to reduce the reader's cognitive load... ie not to force them to read the whimsy if they don't want to :) So how about reordering to:

"Information on Corda Enterprise 4.0 is available: 
`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_.  (Normally this document would have started with a comment about whether or not you'd been recently domiciled under some solidified mineral material
regarding the release of Corda Enterprise 4.0. Alas, we made that joke when we shipped the first release of Corda after Enterprise 3.0 shipped, so the thunder has been stolen and repeating ourselves would be terribly gauche.) "

Comment (fenryka) [reply docs/source/release-notes.rst]: done!

File: docs/source/release-notes.rst [hunk 10]
```diff
@@ -9,11 +9,10 @@ Its been a little under 3 1/2 months since the release of Corda 4.0 and all of t
 of tools Corda offers. Now, following the release of Corda Enterprise 4.0, we are proud to release Corda 4.1, bringing over 150 fixes
 and documentation updates to bring stability and quality of life improvements to those developing on the Corda platform.
 
-Normally this document would have started with a comment about whether or not you'd been recently domiciled under some solidified mineral material
-regarding the release of Corda Enterprise 4.0. Alas, we made that joke when we shipped the first release of Corda after Enterprise 3.0 shipped,
-so the thunder has been stolen and repeating ourselves would be terribly gauche. For those not aware this event occurred, lots of information can be found
-`here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
-`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_.
+Information on Corda Enterprise 4.0 can be found `here <https://www.r3.com/wp-content/uploads/2019/05/CordaEnterprise4_Enhancements_FS.pdf`>_ and
+`here <https://docs.corda.r3.com/releases/4.0/release-notes.html>`_. (It's worth noting that ormally this document would have started with a comment
```
Comment (gendal) [inline docs/source/release-notes.rst]: normally

**Review by gendal — APPROVED** [review]

**Pull request merged**